pkgs/by-name/vm/vmctl/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  bash,
+  shellcheck,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
```

By the way, you can probably get by without a C toolchain here.

```suggestion
  stdenvNoCC,
  bash,
  shellcheck,
  fetchFromGitHub,
}:

stdenvNoCC.mkDerivation {
```